### ISSUE TITLE
Submit via out-of-form element does not contribute the element name/value pair

### ISSUE THREAD

**Issue by mordae** [unpoly/unpoly#494]
When a form is submitted via out-of-form element, such as a `<button form="some-form" name="action" value="add">`, the name/value pair does not make it to the HTTP request.

This happens because `submittingButton` is defined to look for the submitting element only within the confines of the respective `<form>`:

```js
function submittingButton(form) {
    const selector = submitButtonSelector()
    const focusedElement = up.viewport.focusedElementWithin(form)
    if (focusedElement && focusedElement.matches(selector)) {
      return focusedElement
    } else {
      // If no button is focused, we assume the first button in the form.
      return e.get(form, selector)
    }
  }
```

It is necessary to not only do that, but also check if the `activeElement` contains the `form` attribute pointing to the submitted `<form>` and accept it when it does.

**Comment by mordae** [comment]
Hmm, this seems to sometimes not work correctly in Safari.

**Comment by triskweline** [comment]
The fix has been released as part of version 3.2.1.
